@types/karma broken with removal of log4js typings

- [x] I tried using the `@types/karma` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @tkrotoff 

#20518 had removed the log4js typings from DT, as they are now on the log4js repo; however, Karma's typings are now broken as log4js does not export `AppenderConfigBase`:

```
Namespace '"/path/to/project/node_modules/log4js/index"' has no exported member 'AppenderConfigBase'
```

+1. 
I'm facing this issue too. 
```
(286,26): error TS2694: Namespace '"node_modules/log4js/index"' has no exported member 'AppenderConfigBase'.
```

--- a/types/karma/index.d.ts
+++ b/types/karma/index.d.ts
@@ -9,7 +9,6 @@
 // See Karma public API https://karma-runner.github.io/0.13/dev/public-api.html
 import Promise = require('bluebird');
 import https = require('https');
-import log4js = require('log4js');
 
 declare namespace karma {
     interface Karma {
@@ -142,6 +141,16 @@ declare namespace karma {
 
     interface ConfigFile {
         configFile: string;
+    }
+
+    // taken from log4js 1.x typings which are gone...
+    interface Log4jsAppenderConfigBase {
+        type: string;
+        category?: string;
+        layout?: {
+            type: string;
+            [key: string]: any
+        }
     }
 
     interface ConfigOptions {
@@ -283,7 +292,7 @@ declare namespace karma {
          * @default [{type: 'console'}]
          * @description A list of log appenders to be used. See the documentation for [log4js] for more information.
          */
-        loggers?: log4js.AppenderConfigBase[];
+        loggers?: Log4jsAppenderConfigBase[];
         /**
          * @default {}
          * @description Redefine default mapping from file extensions to MIME-type.

